lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -133,34 +142,32 @@ def configure(conf)
           fully_qualified_zone = fetch_gce_metadata('instance/zone')
           @zone = fully_qualified_zone.rpartition('/')[2]
         end
-        if @vm_id.nil?
-          @vm_id = fetch_gce_metadata('instance/id')
-        end
+        @vm_id = fetch_gce_metadata('instance/id') if @vm_id.nil?
       when Platform::EC2
         metadata = fetch_ec2_metadata
-        if @zone.nil? && metadata.has_key?('availabilityZone')
+        if @zone.nil? && metadata.key?('availabilityZone')
```

Comment:
I don't know how to read this: http://ruby-doc.org/core-2.2.2/Hash.html
According to the descriptions there are 4 methods to check for key presence: "has_key", "key", "include", "member" but all examples show "has_key".
Does that mean this is the preferred method? We use "has_hey" here and "include" a little down.